# fingermilletblastgenes
This repository contains genomic and transcriptomic sequences of finger millet blast disease resistance and primer alignments designed to isolate finger millet blast disease resistance RGA orthologs
Blast disease, caused by the fungal pathogen Pyricularia grisea (Magnaporthe grisea) is one of the most common diseases found in many cereal crops, including finger millet. blast disease resistance (R) genes, the knowledge on blast R genes of finger millet, is limited. A comparative genomics study was conducted to identify blast Resistance GeneAanalogs (RGAs) of finger millet using genetic and genomic infortaion including r genes from rice . The syntenic relationships between crop species in the Poaceae family serve as a bridge to retrieve disease resistance gene clusters in closely related crops.Quantitative trait loci (QTL) and meta-QTL associated with resistance to blast disease in rice and other diseases were identified though a bibliography search. FASTA sequences of the QTLs and meta-QTLs, were obtained from the NCBI (https://www.ncbi.nlm.nih.gov).FASTA sequences of the QTLs and meta-QTLs, were obtained from the NCBI (https://www.ncbi.nlm.nih.gov) and were used to perform a BLAST search, in Rice Genome Browser (http://rice.plantbiology.msu.edu/) at threshold 1e-5 without using any filters and using default parameters to identify resistance gene analogues (RGAs).The cDNA sequences of the rice disease resistance genes (R genes) and RGAs were obtained from the Rice Genome Browser (http://rice.plantbiology.msu.edu/), and used as query sequences in a BLAST search in  Comparative Genomics platform (CoGe) (https://genomevolution.org/coge) to retrieve finger millet RGA orthologs, at threshold level 1e-5 and using default expect.Predicted cDNA of the finger millet RGA orthologs were used to query finger millet transcriptome datasets available under the study Metagenome sequencing of blast disease susceptible and resistant finger millet varieties-Raw reads( PRJNA383952,  accession numbers SRX2754533 and SRX2754532) in NCBI SRA databases. In the transcriptome database, paired end Illumina library has been constructed from genomic DNA isolated and pooled from leaf, stem and root of 21 days old seedling. Genomic and transcriptomic sequences of orthologous finger millet RGAs were aligned to rice genomic and  cDNA sequences using “Geneious alignment” algorithm in Geneious prime 2019 4.0.Finger millet RGA orthologs predicted for rice RGAs LOC_Os01g72680 (P680), LOC_Os02g57310 (P310), LOC_Os06g29810 (P810) , LOC_Os11g38480 (P480), LOC_Os11g11960 (P960), LOC_Os11g12330 (P330), LOC_Os11g12340 (P340), LOC_Os11g16510 (P510), LOC_Os11g41170 (P170), LOC_Os11g12350 (P350) were selected for amplification based on the percentage of the length of the rice query cDNA aligning with the predicted finger millet RGA ortholog and the presence of transcriptome evidence. Primers were designed manually and the quality was tested using Net primer (https://www.premierbiosoft.com/netprimer/) .
